Makefile

```diff
@@ -185,3 +192,6 @@ vet: ## Runs go vet
 format: ## format the source
 	${GOFMT} -s -e -l -w .
 	${GO} run golang.org/x/tools/cmd/goimports -l -w .
+
+./providers/terraform-provider-csbpg/cloudfoundry.org/cloud-service-broker/csbpg/1.0.0/linux_amd64/terraform-provider-csbpg_v1.0.0:
```

Comment:
We should use `csbpg/*/linux_amd64/terraform-provider-csbpg_v*` rather than hardcoding the version